### ISSUE TITLE
Alarm for dashboard?

### ISSUE THREAD

**Issue by kfriedman** [NYPL/staff-picks#142]
Should we add an alarm ⏰  for the app to our Alarms Dashboard (as per: https://github.com/NYPL/engineering-general/blob/master/standards/alerting.md)?

Hmm... maybe we need to think about how to get this in our standard production launch workflow.

**Comment by rhernand3z** [comment]
Yes I was going to bring this up during standup. We should add an alarm, not sure how they did this in the Discovery UI? I don't think we are logging on the App, maybe just checking status code responses? Thoughts @EdwinGuzman ?

**Comment by EdwinGuzman** [comment]
Agree. I'll chat to Kang about how he did it for Discovery and then we can implement for Staff Picks. I'll create a ticket to first use the nypl logging standard.

**Comment by holingpoon** [comment]
This may cross reference to Jira ticket [WWW-213](https://jira.nypl.org/browse/WWW-213)

**Comment by holingpoon** [comment]
I don't think the current logging supports me to look for `{ $.level = "ERROR" }` on CloudWatch. I can do a more generic filtering and look where all instances of `Error` occurred on `/aws/elasticbeanstalk/staff-picks-production/var/log/nodejs/nodejs.log`. Any other ideas? cc @ktp242 @gkallenberg

**Comment by EdwinGuzman** [comment]
I believe we should first get logging to work and then we can work on alarms for CW.

**Comment by holingpoon** [comment]
There is a test widget on the Alarm Dashboard at the moment StaffPicksQA, will swap out once the app is on production.
